Examples/Examples/FormViewExampleView.swift

```diff
@@ -16,6 +16,9 @@ import ArcGISToolkit
 import SwiftUI
 
 struct FormViewExampleView: View {
+    /// A Boolean value indicating whether the alert confirming the user's intent to cancel is displayed.
+    @State private var cancelConfirmationIsShowing = false
```

Comment:
To match the existing `isPresented` and the alert argument label (also `isPresented`), maybe follow that form here:
```suggestion
    @State private var isCancelConfirmationPresented = false
```